permission_handler/lib/permission_handler.dart

```diff
@@ -21,6 +23,60 @@ Future<bool> openAppSettings() => _handler.openAppSettings();
 
 /// Actions that can be executed on a permission.
 extension PermissionActions on Permission {
+  /// Callback for when permission is denied.
+  static FutureOr<void>? Function()? onDenied;
```

Comment:
I think we should make these fields private (by starting their names with an `_`), as they are accessed only from internal implementation and should not be exposed outside of the extension.

Reply:
Sounds like a good idea.